yarn-project/aztec/src/sandbox/banana_fpc.ts

```diff
@@ -62,22 +62,22 @@ export async function setupBananaFPC(initialAccounts: InitialAccountData[], wall
   log(`FPC: ${fpc.address}`);
 }
 
-export async function getDeployedBananaCoinAddress(wallet: TestWallet) {
+export async function registerDeployedBananaCoinInWalletAndGetAddress(wallet: TestWallet) {
```

Comment:
using here TestWallet seems incorrect but we don't have `getContracts` on the `Wallet` interface. Shall we expose it there? (or smt. like `isContractRegistered`?)